src/app/Command.h

```diff
@@ -92,13 +93,14 @@ class Command
      *  instance.
      *
      *  @param[in]    apExchangeMgr    A pointer to the ExchangeManager object.
+     *  @param[in]    apDelegate       InteractionModelDelegate set by application.
```

Comment:
delete these parameter comments - they provide no information beyond what the code already reads.

Reply:
Agree, would like to get a style agreement regarding doxygen style, then I would cleanup all params in IM

Reply:
https://github.com/project-chip/connectedhomeip/issues/5624

Reply:
sure, would address it using #5624